src/main/webapp/features/screens/game/dashboard/chooseResources/ct-choose-resources.js

```diff
@@ -20,6 +22,9 @@ angular.module('catan')
             function init(scope, type) {
 
                 var playerResources = scope.game.getCurrentUser().resources;
+                var tradePortRatio = (type === "TRADE_PORT") ? scope.game.getCurrentUser().availableActions.list.filter(function(item) {
+                    return item.code === "TRADE_PORT";
+                })[0].params : {};
```

Comment:
I prefer to write such operator in the following way:
var tradePortRatio = (type === "TRADE_PORT")
                    ? scope.game.getCurrentUser().availableActions.list.filter(function(item) { return item.code === "TRADE_PORT"; })[0].params
                    : {};